app/controllers/scorecards/request_changes_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Scorecards
+  class RequestChangesController < ApplicationController
+    before_action :set_scorecard
+    before_action :find_request_change, only: [:edit, :update, :approve, :reject]
+
+    def new
+      @request_change = authorize @scorecard.request_changes.new
+    end
+
+    def edit
+    end
+
+    def create
+      @request_change = authorize @scorecard.request_changes.new(request_change_params)
+
+      if @request_change.save
+        flash[:notice] = t("scorecard.create_successfully")
+        redirect_to scorecard_url(@scorecard.uuid)
+      else
+        render :new
+      end
+    end
+
+    def update
+      if @request_change.update(request_change_params)
```

Comment:
@hengsokly please check the authorization for this, :update, and :approve actions

Reply:
I already check it in find_request_change

def find_request_change
    @request_change = authorize @scorecard.request_changes.find(params[:id])
end